src/ReactWrapperComponent.jsx

```diff
@@ -32,9 +32,9 @@ export default function createWrapperComponent(node, options = {}) {
       };
     },
 
-    setChildProps(newProps) {
+    setChildProps(newProps, cb = undefined) {
```

Comment:
Function lengths are intended to only count required arguments - the language follows this for the most part. You're right that React doesn't make this explicit, but I'd prefer to correct their mistake for our API.